webapp/channels/src/components/threading/global_threads/thread_item/thread_item.tsx

```diff
@@ -188,130 +195,148 @@ function ThreadItem({
+                            >
+                                <DotsVerticalIcon size={18}/>
+                            </Button>
+                        </WithTooltip>
+                    </ThreadMenu>
+                </div>
+
+                {/* The strange interaction here where we need a click/keydown handler messes with the ESLint rules, so we just disable it */}
+                {/*eslint-disable-next-line jsx-a11y/no-static-element-interactions*/}
+                <div
+                    className='preview'
+                    dir='auto'
+                    onClick={handleFormattedTextClick}
+                    onKeyDown={handleFormattedTextClick}
+                >
+                    {post.message ? (
+                        <Markdown
+                            message={post.state === Posts.POST_DELETED ? msgDeleted : post.message}
+                            options={markdownPreviewOptions}
+                            imagesMetadata={post?.metadata && post?.metadata?.images}
+                            mentionKeys={mentionsKeys}
+                            imageProps={imageProps}
                         />
+                    ) : (
+                        <Attachment post={post}/>
                     )}
-                    {isPostPriorityEnabled && (
-                        thread.is_urgent && (
-                            <PriorityBadge
-                                className={postAuthor === channel?.display_name ? 'ml-2' : ''}
-                                priority={PostPriority.URGENT}
-                            />
-                        )
+                </div>
+                <div className='activity'>
+                    {participantIds?.length ? (
+                        <Avatars
+                            userIds={participantIds}
+                            size='xs'
+                        />
+                    ) : null}
+                    {Boolean(totalReplies) && (
+                        <>
+                            {newReplies ? (
+                                <FormattedMessage
+                                    id='threading.numNewReplies'
+                                    defaultMessage='{newReplies, plural, =1 {# new reply} other {# new replies}}'
+                                    values={{newReplies}}
+                                />
+                            ) : (
+                                <FormattedMessage
+                                    id='threading.numReplies'
+                                    defaultMessage='{totalReplies, plural, =0 {Reply} =1 {# reply} other {# replies}}'
+                                    values={{totalReplies}}
+                                />
+                            )}
+                        </>
                     )}
                 </div>
+                {showListTutorialTip && isFirstThreadInList && isMobileView && (<CRTListTutorialTip/>)}
+            </div>
+            <span
```

Comment:
Having this out here seems a bit weird. I don't suppose it's possible to have this inside the div now that we have the `aria-label` on it preventing the screen reader from reading out the entire content of the thread item?

Reply:
Yep that seems to work. Will move inside the div.